.env.example

```diff
@@ -19,3 +19,7 @@
 # Set this only when running in Azure Container Apps environment
 # RUNNING_IN_PRODUCTION=1
 
+# Locally set this to a random string
+# Remotely set this with the following command:
+# python -c 'import secrets; print(secrets.token_hex())'
+# SECRET_KEY=<secret key>
```

Comment:
Same comment as in other repo:

Interesting, this sample puts both local and prod settings in the same env file, but prefixed. That doesn't feel like a typical thing to do (especially as you'd be storing production secrets in plain text locally!), but given that's the case, it might be more appropriate to have an AZURE_SECRET_KEY and LOCAL_SECRET_KEY, and then getenv() for each of those in the appropriate settings files. LOCAL_SECRET_KEY could either go in settings.py or development.py, and AZURE_SECRET_KEY would be in production.py.